README.md

```diff
@@ -1,6 +1,22 @@
 ZMTP Wireshark Dissector
 ========================
 
+*THIS LUA DISSECTOR WILL BE OBSOLETED!!!!*
+
+*A built-in dissector, written in C, has been merged into master in April 2024, see
+https://gitlab.com/wireshark/wireshark/-/merge_requests/14947.  It will mostly likely
+become part of an official release towards the end of 2024.*
```

Comment:
Would it be better to have an MR now that only notes the upcoming obsolescence, then push another one later when it is included in a major release?  That way, the inconvenience of the renaming will not happen for most people before it has to.